src/Console/MutationMakeCommand.php

```diff
@@ -8,78 +8,35 @@
 
 class MutationMakeCommand extends GeneratorCommand
 {
-    /**
-     * The name and signature of the console command.
-     *
-     * @var string
-     */
     protected $signature = 'make:graphql:mutation {name}';
-    /**
-     * The console command description.
-     *
-     * @var string
-     */
     protected $description = 'Create a new GraphQL mutation class';
-    /**
-     * The type of class being generated.
-     *
-     * @var string
-     */
     protected $type = 'Mutation';
 
-    /**
-     * Get the stub file for the generator.
-     *
-     * @return string
-     */
     protected function getStub()
     {
         return __DIR__.'/stubs/mutation.stub';
     }
 
-    /**
-     * Get the default namespace for the class.
-     *
-     * @param string $rootNamespace
-     *
-     * @return string
-     */
     protected function getDefaultNamespace($rootNamespace)
     {
-        return $rootNamespace.'\GraphQL\Mutation';
+        return $rootNamespace.'\GraphQL\Mutations';
     }
 
-    /**
-     * Build the class with the given name.
-     *
-     * @param string $name
-     *
-     * @return string
-     */
     protected function buildClass($name)
     {
         $stub = parent::buildClass($name);
 
-        return $this->replaceType($stub, $name);
+        return $this->replaceGraphqlName($stub);
```

Comment:
Use adapted code to it's generated with a lower case start character, per general GraphQL convention.